src/resources/postcss/tickets/_modal.pcss

```diff
@@ -0,0 +1,314 @@
+				background-size: contain;
+				border-radius: 50%;
+				height: var(--spacer-4);
+				width: var(--spacer-4);
+
+				&:hover,
+				&:focus {
+					background-image: svg-inline(remove);
+					background-position: center center;
+					background-repeat: no-repeat;
+					background-size: contain;
+				}
+			}
+
+			.tribe-tickets__item__content__title {
+				@mixin heading-8;
+
+				-ms-grid-column: 1;
+				grid-column: 1;
+				-ms-grid-row: 1;
+				grid-row: 1;
+
+				@media(--viewport-medium) {
+					@mixin desktop-heading-6;
+
+					-ms-grid-column: 2;
+					grid-column: 2;
+				}
+
+				&.tribe-tickets--no-description {
+					align-self: center;
+					-ms-grid-row: 1;
+					-ms-grid-row-span: 2;
+					grid-row: 1 / span 2;
+					padding-top: 0;
+				}
+			}
+
+			.tribe-tickets__item__extra {
+				align-self: center;
+				display: flex;
+				flex-flow: column;
+				-ms-grid-column: 2;
+				grid-column: 2;
+				-ms-grid-row: 1;
+				-ms-grid-row-span: 2;
+				grid-row: 1 /span 2;
+				-ms-grid-columns:  auto;
+				grid-template-columns: auto;
+				-ms-grid-rows: auto;
+				grid-template-rows: auto;
+				text-align: center;
+
+				@media(--viewport-medium) {
+					-ms-grid-column: 3;
+					grid-column: 3;
+					margin-right: var(--spacer-1);
+				}
+
+				&.tribe-tickets__item__extra--price-suffix {
+					@media(--viewport-medium) {
+						display: inline-block;
+					}
+				}
+			}
+
+			.tribe-tickets__item__extra__price {
+				@mixin mobile-body-1;
+
+				@media (--viewport-medium) {
+					@mixin desktop-body-1-sup;
+				}
+			}
+
+			.tribe-tickets__item__extra__available {
+				@mixin hidden;
+			}
+
+			.tribe-tickets__item__details__summary {
+				-ms-grid-column: 1;
+				grid-column: 1;
+				-ms-grid-row: 2;
+				grid-row: 2;
+				padding-left: 0;
+
+				@media(--viewport-medium) {
+					-ms-grid-column: 2;
+					grid-column: 2;
+				}
+			}
+
+			.tribe-tickets__item__details__content {
+				-ms-grid-column: 1;
+				-ms-grid-column-span: 4;
+				grid-column: 1 / span 4;
+				-ms-grid-row: 3;
+				grid-row: 3;
+				margin: var(--spacer-0 ) 0;
+
+				@media(--viewport-medium) {
+					-ms-grid-column: 2;
+					-ms-grid-column-span: 2;
+					grid-column: 2 / span 2;
+				}
+			}
+
+			.tribe-tickets__item__quantity {
+				@mixin heading;
+				@mixin mobile-heading-6;
+
+				flex: 0 0 64px;
+				-ms-grid-column: 3;
+				grid-column: 3;
+				-ms-grid-row: 1;
+				-ms-grid-row-span: 2;
+				grid-row: 1 / span 2;
+
+				@media(--viewport-medium) {
+					@mixin desktop-heading-6;
+
+					-ms-grid-column: 4;
+					grid-column: 4;
+				}
+			}
+
+			.tribe-tickets__item__quantity__number {
+				@mixin mobile-body-2;
+				border: 1px solid var(--color-border-default);
+				border-radius: var(--border-radius-default);
+				height: 40px;
+				margin: 0 var(--spacer-2);
+				width: 44px;
+
+				@media(--viewport-medium) {
+					@mixin desktop-body-2-sup;
+					border: 0;
+					height: 48px;
+					margin: 0;
+					width: 30px;
+				}
+			}
```

Comment:
if you add `tribe-common-b2` class to this element, you won't need the mixins.

Reply:
I've been trying to trace where these things are taking place. 

Apparently these mods are like this because it's using the markup of the tickets block, from the modal and need adjustments in the font sizes. 

So, these elements already have `tribe-common-b2`, `tribe-common-b3` but the size should be different for **the modal**.

I'm afraid that it'll have to stay like this.

Reply:
👍